### PR TITLE
Update typetalk.py

### PR DESCRIPTION
**Pull request by is2ei**
+label: docsite_pr

##### SUMMARY
Typetalk usess "com" domain now.

##### ISSUE TYPE
- Bugfix Pull Request

##### SUMMARY
Now Typetalk uses "com" domain.
FYI: https://www.typetalk.com/blog/typetalk-gets-new-top-level-domain-typetalk-com/

##### COMPONENT NAME
typetalk

### CODE REVIEW

File: lib/ansible/modules/notification/typetalk.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 version_added: "1.6"
 short_description: Send a message to typetalk
 description:
-  - Send a message to typetalk using typetalk API ( http://developers.typetalk.in/ )
+  - Send a message to typetalk using typetalk API ( https://developer.nulab-inc.com/docs/typetalk/ )
```
Comment (dagwieers) [inline lib/ansible/modules/notification/typetalk.py]: Is there any relevance in having the URL in the module's description ?
I would move it into the notes and make it a proper link.
Use either U(https://developer.nulab-inc.com/docs/typetalk/) or L(Typetalk API,https://developer.nulab-inc.com/docs/typetalk/).

**Review by dagwieers — CHANGES_REQUESTED** [review]

**Comment by is2ei** [comment]
@dagwieers 
Thanks for your comment! I updated it. Could you review it?

**Comment by dagwieers** [comment]
@is2ei Why did you close this ?

**Comment by is2ei** [comment]
@dagwieers 
Thanks for your comment. I created another PR. 
https://github.com/ansible/ansible/pull/46045

**Pull request closed without merging**